.github/workflows/vendor_gardener.yaml

```diff
@@ -0,0 +1,35 @@
+name: Vendor Gardener
+on:
+  push:
+    branches:
+    - dependabot/go_modules/github.com/gardener/**
+permissions: write-all
+jobs:
+  run:
+    name: Run make revendor
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version-file: go.mod
+
+    - name: Use gardener's replaced client-go version
+      run: go mod edit -replace 'k8s.io/client-go=k8s.io/client-go@'"$(go list -m -f '{{.Version}}' k8s.io/api)"
+
```

Comment:
/hold
Also, some change is required in the workflow as the current config is failing.
```suggestion
```

Reply:
Good catch, thanks! After https://github.com/gardener/gardener-extension-registry-cache/pull/29 we don't need this step.

Reply:
/unhold

Reply:
/hold cancel